dags/dbt_manual.py

```diff
@@ -0,0 +1,35 @@
+import airflow
+from airflow.decorators import task
+from airflow.models.param import Param
+from airflow.operators import bash
+
+from dags.common import db, dbt, default_dag_args, slack
+
+
+dag_args = default_dag_args() | {"default_args": dbt.get_default_args()}
+
+with airflow.DAG(
+    dag_id="dbt_manual",
+    schedule=None,
+    params={
+        "manual_dbt_run_args": Param(
+            "", type="string", description="write dbt run arguments, for example : --select stg_reseaux+"
```

Comment:
Je dirais pas de défaut pour éviter de lancer un `dbt run` sans argument par erreur. 
```suggestion
            type="string", description="write dbt run arguments, for example : --select stg_reseaux+"
```